src/content/stakeholders/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Key Players in the Process
-description: "The important participants in the GSA SmartPay program and descriptions of their roles"
+description: The important participants in the GSA SmartPay program and descriptions of their roles
```

Comment:
Learn more about the key players in the GSA SmartPay program with a description of their roles and responsibilities.